### PR TITLE
fixes plurality of error message

### LINKED ISSUE #1377: Plurality of error messages
### Expected behaviour
When there is only one error message, it should be worded appropriately, something like:
- This page has an error
- This must be corrected to continue

### Actual behaviour
When only one error is raised the wording is:
- This page has 1 errors 
- These must be corrected to continue

### Steps to reproduce the behaviour
On any screen that can generate an error, input invalid data and submit.

### Screenshot
image

### PR DESCRIPTION
**Pull request by yoganandkunche**
### What is the context of this PR?
Fixes #1377 

### How to review 
Retest #1377 

And all tests should pass

### CODE REVIEW

File: app/templates/questionnaire.html [hunk 1]
```diff
@@ -7,7 +7,11 @@
 
     <div class="panel panel--error u-mb-s" data-qa="error">
       <div class="panel__header">
-        <h1 class="panel__title venus">This page has {{mapped_errors|length}} errors</h1>
+          {% if mapped_errors|length > 1 %}
+            <h1 class="panel__title venus">This page has {{mapped_errors|length}} errors</h1>
+          {%- else -%}
+            <h1 class="panel__title venus">This page has an error</h1>
+          {%- endif -%}
```
Comment (dwyeradam) [inline app/templates/questionnaire.html]: We should discuss the use of logic in HTML.

File: app/templates/questionnaire.html [hunk 2]
```diff
@@ -7,7 +7,11 @@
...
       </div>
       <div class="panel__body" data-qa="error-body">
         <p class="mars">These <strong>must be corrected</strong> to continue.</p>
```
Comment (dwyeradam) [inline app/templates/questionnaire.html]: Shouldn't "These" be "This" for a single error?

**Review by dwyeradam — CHANGES_REQUESTED** [review]

File: app/templates/questionnaire.html (hunk 2 above)
Comment (dwyeradam) [inline app/templates/questionnaire.html]: Try this from flask babel:
{{ ngettext('This', 'These', mapped_errors|length) }} <strong>must be corrected</strong> to continue.

**Review by dwyeradam — CHANGES_REQUESTED** [review]

File: tests/integration/star_wars/test_errors.py [hunk 3]
```diff
@@ -41,4 +41,18 @@ def test_skip_question_errors(self):
 
         # Then errors exists on page
         self.assertStatusOK()
-        self.assertInPage('This page has 1 errors')
+        self.assertInPage('This page has an error')
+        self.assertInPage('This <strong>must be corrected</strong> to continue.')
```
Comment (dwyeradam) [inline tests/integration/star_wars/test_errors.py]: It may be worth adding a test for multiple errors showing "These" as well.

**Review by shahi645 — APPROVED** [review]
@yoganandkunche Looks good!

File: tests/integration/star_wars/test_errors.py (hunk 3 above)
Comment (shahi645) [reply tests/integration/star_wars/test_errors.py]: @dwyeradam This is covered below see line 58

**Review by dwyeradam — APPROVED** [review]

**Comment by dwyeradam** [comment]
@yoganandkunche Squishy Squashy

**Pull request merged**